contracts/test/TestExpiryAccount.sol

```diff
@@ -22,17 +22,18 @@ contract TestExpiryAccount is SimpleAccount {
         addTemporaryOwner(anOwner, type(uint256).max);
     }
 
-    function addTemporaryOwner(address owner, uint256 deadline) public onlyOwner {
-        ownerDeadlines[owner] = deadline;
+    function addTemporaryOwner(address owner, uint256 sigTimeRange) public onlyOwner {
+        ownerDeadlines[owner] = sigTimeRange;
```

Comment:
It's no longer ownerDeadlines, but time range.

Reply:
yes, this test only checks for deadline, not for validAfter, so ownerDeadlines is ok, but sigTimeRange is wrong (it should be "deadline"
another test will check for validAfter, since it is not this use-case